### DISCUSSION TITLE
How to upgrade to mozc "2.26.4472.100"?

### DISCUSSION THREAD

**Discussion by bertsons** [google/mozc#901] [Q&A] [answered]
I am having the issue from here https://bugs.launchpad.net/ubuntu/+source/mozc/+bug/1931554 , which was apparently fixed in version 2.26.4472.100 of mozc. I currently have a lower version than this on Ubuntu 22.04. Since the latest versions of mozc don't appear in the Ubuntu repositories, I can't update it the usual way. What is the easiest way for me to do this? FYI my current version is 2.26.4220.100

**Comment by hiroyuki-komatsu** [comment, marked as answer]
You can find a zip file with binaries in the GitHub Actions page.
* https://github.com/google/mozc/actions/workflows/linux.yaml

It seems there are also some articles about installing new Mozc for Ubuntu 22.04 (in Japanese).
* https://zenn.dev/ikuya/articles/aa69fd1009b773
* https://zenn.dev/phoepsilonix/articles/0c492a22a3c9d0
